lib/mod/cache/fetch.go

```diff
@@ -66,17 +134,17 @@ func fetchGitHub(lang, owner, repo, tag string) (err error) {
 	}
 
 	/*
```

Comment:
Might as well remove the commented out section rather than reformatting it

Reply:
This is go fmt doing its magic, and I didn't want to remove a commented section I did not write. Wilco